nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -360,6 +361,15 @@ in
         '';
       };
 
+      systemFeatures = mkOption {
```

Comment:
I'd probably name it `supportedFeatures` to make it compatible the option from `nix.machines`.

Reply:
But it is ```system-features``` in ```/etc/nix/nix.conf```
So the naming is already inconsistent